README.md

```diff
@@ -96,14 +97,17 @@ folders for you during the build.
 
 ## Alternatives
 
-As `Cabal` does not currently provide the feature of automatic modules
-discovery, there are some workarounds for this process. One of them is this
-library — `autopack` that uses `Cabal`'s Setup feature to discover all modules
-during the pre-build stage of `Cabal`. Another tool that provides it
-out-of-the-box is [`hpack`](https://github.com/sol/hpack). It is the wrapper on
```

Comment:
I would prefer not to remove this part of text.
I would rather add a sentence about `cabal-fmt` in the end of paragraph the same style that `autopack` and `hpack`.